arch/arm/src/samv7/sam_freerun.c

```diff
@@ -220,8 +220,8 @@ int sam_freerun_initialize(struct sam_freerun_s *freerun, int chan,
 int sam_freerun_counter(struct sam_freerun_s *freerun, struct timespec *ts)
 {
   uint64_t usec;
-  uint32_t counter;
-  uint32_t verify;
+  uint16_t counter;
```

Comment:
@pkarashchenko since you are changing "counter" and "verify" to uint16_t I suggest you to move down after the uint32_t variables. It is not a requirement, but a good practice to avoid alignment issues.

Reply:
In this particular case it does not matter because 2 of uint16_t are exactly uint32_t size, but I will move.